src/ancillary/__tests__/anchor-request-params-parser.test.ts

```diff
@@ -117,4 +128,9 @@ describe('AnchoRequestParamsParser', () => {
     expect(params.capCID).toEqual(FAKE_CAPCID)
   })
 
+  test('throws error on invalid car file', () => {
+    const validation =  parser.parse(CAR_FILE_INVALID as ExpReq)
+    console.log("Here is invalid validation: " + JSON.stringify(validation))
+    expect(() => parser.parse(CAR_FILE_INVALID as ExpReq)).toThrow(/^Can not decode CAR file/);
```

Comment:
so, this doesn't throw an error, on the junk carfile.  instead it happily extracts this stuffs

```
Here is invalid validation: {"_tag":"Left","left":[{"trail":[{"key":"","type":{"name":"RequestAnchorParamsV2"},"actual":{"0":67,"1":114,"2":101,"3":97,"4":116,"5":101,"6":100,"7":66,"8":121,"9":67,"10":104,"11":97,"12":116,"13":71,"14":80,"15":84,"16":52,"17":89,"18":111,"19":117,"20":99,"21":97,"22":110,"23":85,"24":115,"25":101,"26":84,"27":104,"28":105,"29":115,"30":83,"31":116,"32":114,"33":105,"34":110,"35":103}}]}]}
```

Reply:
@ukstv i moved it here so i could merge the other one, since this is on some old existing code